Core/src/com/serotonin/m2m2/db/dao/pointvalue/BoundaryAggregateDao.java

```diff
@@ -26,6 +27,12 @@ public interface BoundaryAggregateDao extends AggregateDao {
      */
     ZonedDateTime boundary();
 
+    /**
+     * @param unit TemporalUnit
+     * @return value of requested unit from boundary
+     */
+    long fromBoundary(TemporalUnit unit);
```

Comment:
We discussed returning a `TemporalAmount`, why did you not do it this way? Also since this is an API change we can't merge this into a 4.4.x, do you have a plan for what to do for 4.4.x?

Reply:
I couldn't find a way to add 2 temporal amounts (overlap and boundary). Looks like new delta like rollup type also requires API change and we need to release 4.5.0 eventually.